airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -83,7 +83,7 @@ class AWSBatchOperator(BaseOperator):
     template_fields = ('job_name', 'overrides',)
 
     @apply_defaults
-    def __init__(self, job_name, job_definition, job_queue, overrides, array_properties=None,
+    def __init__(self, job_name, job_definition, job_queue, overrides, array_properties={},
```

Comment:
You shouldn't do it like this. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Instead in the method do

```
        self.array_properties = array_properties if array_properties is not None else {}
```